pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: workmanager
 description: Flutter Workmanager. This plugin allows you to schedule background work on Android and iOS.
-version: 0.5.0-dev.8
+version: 0.5.0-dev.8-enfold
```

Comment:
Can you please remove this change?

Reply:
I've gone ahead and reverted this change